css-pseudo-4/Overview.bs

```diff
@@ -758,6 +777,60 @@ Painting the Highlight</h3>
     css/css-pseudo/highlight-painting-003.html
     </wpt>
 
+<h4 id="paired-cascade">
+Highlight Colors</h4>
```

Comment:
I think this section can also be simplified a bit, but, I'll have to take a pass at it later. :)

Reply:
by all means!

Reply:
i’ve rewritten and heavily simplified this section in 86bb487445f17b354230b7832e116e697982d6de.